src/fn/upload-custom-translations.js

```diff
@@ -42,12 +42,8 @@ function propertiesAsObject(path) {
   return vals;
 }
 
-function mergeProperties(doc, props) {
-  if(!doc.values) doc.values = {};
-
-  for(const k in props) {
-    if(props.hasOwnProperty(k)) doc.values[k] = props[k];
-  }
+function overwriteProperties(doc, props) {
+  doc.custom = props;
```

Comment:
medic-conf can be used on old versions of webapp too - they aren't locked together. So you'll need to make this code backwards compatible somehow, eg:

```js
if(doc.default) {
  // 3.4.0 translation structure
  doc.custom = props;
} else {
  // obsolete doc structure
  if(!doc.values) {
    doc.values = {};
  }
  for(const k in props) {
    if(props.hasOwnProperty(k)) {
      doc.values[k] = props[k];
    }
  }
}
```